### PR TITLE
Adding rules for detection of step 6.A

### LINKED ISSUE #9161: Detect 6.A - Remote System Discovery
Generate detection rules using the Carbanak [step 6.A](https://github.com/center-for-threat-informed-defense/adversary_emulation_library/tree/master/carbanak/Emulation_Plan/Scenario_1#6a---remote-system-discovery-t1018-t1087002)

- [x]  Test environment setup
- [x]  Collect relevant logs
- [ ]  Write appropriate decoders and rules

### Procedures

1. Interact with bankdc Meterpreter session
    ```
    msf > sessions -i 2
    ```

2. Load PowerShell into memory
    ```
    meterpreter > load powershell
    ```
   
3. Execute `Get-ADComputer` against the `cfo` workstation from memory
    ```
    meterpreter > powershell_execute "Get-ADComputer -Identity 'cfo' -Properties *"
    ```
   
4. Import `PowerView` into memory
    ```
    meterpreter > powershell_import /home/<attacker>/carbanak/Resources/step6/powerview.ps1
    ```
   
5. Execute `PowerView`'s `Get-NetUser` from memory
    ```
    meterpreter > powershell_execute Get-NetUser
    ```

### Detection Criteria:

6.A.1: tiny.exe loads system.management.automation.dll
6.A.2: PowerShell executes Get-ADComputer
6.A.3: PowerShell executes Get-NetUser

### PR DESCRIPTION
**Pull request by fabamatic**
Fixing error in powershell rules

|Related issue|
|---|
|closes #9161 |

## Description

Adding few rules for detection of Step 6.A.
Fixing rule dependency errors in 0915-win-powershell_rules.xml

## Configuration options

Some of the rules require Powershell script block capture and collection to be enabled to work

### CODE REVIEW

**Review by JavierBejMen — APPROVED** [review]
LGTM

**Review by jcruzlp — APPROVED** [review]

File: ruleset/rules/0915-win-powershell_rules.xml [hunk 1]
```diff
@@ -19,20 +19,40 @@
...
+        <description>Executed Powershell script "Get-ADComputer" executed</description>
+        <mitre>
+            <id>T1018</id>
+        </mitre>
+    </rule>
+
+    <!-- Sample: {"win":{"eventdata":{"messageNumber":"1","messageTotal":"1","scriptBlockText":"Get-NetUser","scriptBlockId":"b8633c48-c1ea-47b5-bad6-7766ff636abb"},"system":{"eventID":"4104","keywords":"0x0","providerGuid":"{A0C1853B-5C40-4B15-8766-3CF1C58F985A}","level":"5","channel":"Microsoft-Windows-PowerShell/Operational","opcode":"15","message":"\"Creating Scriptblock text (1 of 1):\r\nGet-NetUser\r\n\r\nScriptBlock ID: b8633c48-c1ea-47b5-bad6-7766ff636abb\r\nPath: \"","version":"1","systemTime":"2021-06-30T22:20:57.104893900Z","eventRecordID":"5329","threadID":"9440","computer":"bankdc.ExchangeTest.com","task":"2","processID":"3940","severityValue":"VERBOSE","providerName":"Microsoft-Windows-PowerShell"}}} -->
+    <rule id="91806" level="3">
+        <if_sid>91802</if_sid>
+        <field name="win.eventdata.scriptBlockText" type="pcre2">(?i)Get-NetUser</field>
```
Comment (NicoGuini) [inline ruleset/rules/0915-win-powershell_rules.xml:52]: Why don't use osregex? By default it's case-insensitive

**Review by NicoGuini — COMMENTED** [review]
Comments made

**Review by 72nomada — DISMISSED** [review]
LGTM

File: ruleset/rules/0915-win-powershell_rules.xml (hunk 1 above)
Comment (fabamatic) [reply ruleset/rules/0915-win-powershell_rules.xml:52]: I think the team agreed on suing pcre2

**Review by NicoGuini — APPROVED** [review]
LGTM

**Review by JavierBejMen — APPROVED** [review]
LGTM

**Review by 72nomada — APPROVED** [review]
LGTM

**Pull request merged**